src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -70,8 +74,10 @@ public boolean isVisible()
 	QuantumNetworkBridge( Constants.CATEGORY_NETWORK_FEATURES ),
 	Channels( Constants.CATEGORY_NETWORK_FEATURES ),
 
+	Interface( Constants.CATEGORY_NETWORK_BUSES ),
 	LevelEmitter( Constants.CATEGORY_NETWORK_BUSES ),
 	CraftingTerminal( Constants.CATEGORY_NETWORK_BUSES ),
+	Terminal( Constants.CATEGORY_NETWORK_BUSES ),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)